wigwam/commands.py

```diff
@@ -252,6 +255,64 @@ def cmake_install(tag: str, base: str, no_cache: bool = False) -> Image:
     )
 
 
+def test(
+    tag: str,
+    logfile: os.PathLike[str] | str,
+    compress_output: bool,
+    quiet_fail: bool,
+) -> None:
+    """
+    Run all ctests from the Docker image work directory.
+
+    Parameters
+    ----------
+    tag : str
+        The tag of the image to test.
+    logfile : os.PathLike[str] | str
+        The name of the XML test output file.
+    compress_output : bool
+        If True, compress the output of the test.
+    quiet_fail : bool
+        If True, don't output on failure.
+    """
+
+    prefixed_tag: str = prefix_image_tag(tag)
+
+    host_volume_path = "./Testing/Temps"
```

Comment:
Instead of hard-coding this directory path, let's mount the parent directory of `logfile` in the container.